plugins/trix/src/TrixTextSearchAdapter/TrixTextSearchAdapter.test.ts

```diff
@@ -31,17 +31,17 @@ test('adapter can fetch output files', async () => {
   const results = await adapter.searchIndex({
     queryString: 'apple',
   })
-
   // check results are of type BaseResult for prefix search
   expect(results[0] instanceof BaseResult).toBeTruthy()
   expect(results[0].getLabel()).toEqual('Apple2')
-  expect(results[1].getLabel()).toEqual('Apple3')
+  // now there are 2 results with apple2 and 2 with apple3
+  expect(results[2].getLabel()).toEqual('Apple3')
```

Comment:
can you elaborate on the reason why it changed?

Reply:
Yeah, the addition of the plain text adapter tracks adds an additional entry to both apple2 and apple3 because we are indexing the same file with (gff3adapter and the gff3tabixadapter tracks). The results are coming from the different tracks we indexed, so apple2 and apple3 now has an entry coming from the gff3Adapter track and another from the  gff3tabixadapter track. When we return the results, we now have apple2, apple2, apple3, apple3.

Reply:
sounds good, can you update the comment with this